content/_data/changelogs/weekly.yml

```diff
@@ -1125,6 +1125,40 @@
     # pull 2998 (parent POM) doesn't seem notable enough
     # pull 3000 impact unknown. Just some minor robustness fix?
     # pull 3023 too minor
+- version: "2.80"
+  date: 2017-09-24
+  changes:
+    - type: bug
+      message: >
+        Save the current Jenkins version whenever saving the Jenkins object, e.g. when saving the global security configuration.
+        Plugin may rely on this information for data migration that would be triggered unnecessarily.
+      pull: 3010
+      issue: 42577
+    - type: bug
+      message: >
+        Prevent possible <code>NullPointerException</code> when removing an item from a list view due to a race condition.
+      pull: 3007
+      issue: 23411
+    - type: bug
+      message: >
+        Avoid a possible server-side timeout on long-running CLI commands using plain HTTP mode by sending periodic pings from the client.
+      pull: 3011
+      issue: 46659
+    - type: bug
+      message: >
+        Renaming or moving a folder failed to properly move build directories of its children when using custom build directory, resulting of loss of their builds.
+      pull: 2932
+      issue: 44657
+    - type: rfe
+      message: >
+        Developer: Deprecate <code>TimeUnit2</code> and replace with <code>TimeUnit</code>.
```

Comment:
Should we specify the FQN? I think it's important for at least the replacement one so that there's no doubt? aka `java.util.concurrent.TimeUnit` (and `hudson.util.TimeUnit2` for the current one, but people hitting an issue with it will see it, so less needed IMO)